lib/views/general/_frontpage_requests_list.html.erb

```diff
@@ -1,15 +1,17 @@
 <%- @request_events, @request_events_all_successful = InfoRequest.recent_requests %>
-<div id="examples_1">
+<div id="examples_1" class="bodies-list">
```

Comment:
Do you think it might be better to add a new class 'requests-list' or something, that has the relevant styles, rather than adding a non-semantic class name here? That means if we want to change the styles of the bodies list in the future, we wont accidentally change this list too.